lib/core-integration/src/Test/Integration/Scenario/CLI/Network.hs

```diff
@@ -86,6 +94,12 @@ spec = do
             let maxEpoch = show $ fromIntegral @Word31 @Int maxBound
             params <- getNetworkParamsViaCliExpectingFailure ctx maxEpoch
             params `shouldContain` (errMsg404NoEpochNo maxEpoch)
+
+    it "CLI_NETWORK - network clock" $ \ctx -> do
+        eventually "ntp status = available" $ do
```

Comment:
We need to mark this one as "pendingOnWindows" (see Test.Utils.Windows)

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1428/commits/d17692c6d1b554f325831040151bd8bfe446bb83